ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -271,21 +286,30 @@ open class TwoLineTitleView: UIView, TokenizedControlInternal {
         self.animatesWhenPressed = animatesWhenPressed
         self.accessoryType = accessoryType
 
-        titleButtonLeadingImageView.image = titleImage
-        titleButtonLeadingImageView.isHidden = titleImage == nil
+        titleLeadingImageView.image = titleImage
+        titleLeadingImageView.isHidden = titleImage == nil
+
+        setupTitleLine(titleContainer, label: titleLabel, trailingImageView: titleTrailingImageView, imageSize: TokenSetType.titleImageSizeToken, text: title, interactive: interactivePart.contains(.title), accessoryType: accessoryType)
+        if titleLeadingImageView.image != nil {
+            titleContainer.insertArrangedSubview(titleLeadingImageView, at: 0)
+        }
 
-        setupButton(titleButton, label: titleButtonLabel, trailingImageView: titleButtonTrailingImageView, text: title, interactive: interactivePart.contains(.title), accessoryType: accessoryType)
         // Check for strict equality for the subtitle button's interactivity.
-        // If the whole area is active, we'll stretch the title button to adjust the hit area
-        // while still only keeping one button active from an accessibility standpoint.
-        setupButton(subtitleButton, label: subtitleButtonLabel, trailingImageView: subtitleButtonImageView, text: subtitle, interactive: interactivePart == .subtitle, accessoryType: accessoryType)
+        // If the whole area is active, we'll use the title as the main accessibility item.
+        setupTitleLine(subtitleContainer, label: subtitleLabel, trailingImageView: subtitleImageView, imageSize: TokenSetType.subtitleImageSizeToken, text: subtitle, interactive: interactivePart == .subtitle, accessoryType: accessoryType)
```

Comment:
This should probably be `imageSize: TwoLineTitleViewTokenSet.subtitleImageSizeToken`, same with `titleContainer` above.

Reply:
I'm not convinced. `TwoLineTitleView.TokenSetType` and `TwoLineTitleViewTokenSet` are one and the same, and if you ask me, the fact that `TokenSetType` is shorter makes the code more pleasant to read.

Reply:
We generally prefer to be more explicit about which token set we're referencing, and the `typealias` is primarily intended for use in the shared base class.

Reply:
Third option: omit the type entirely and just write `imageSize: .subtitleImageSizeToken`

Reply:
jk doesn't work in this case

Reply:
The issue about being explicit or implicit about type aliases is a bigger discussion that we can have another time. For now though, this particular instance is no longer applicable since `setupTitleLine` now infers the appropriate size based on whether we're in the title or subtitle line. Marking this as resolved for now.